packages/guides/src/Renderer/RenderSet.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace phpDocumentor\Guides\Renderer;
+
+use phpDocumentor\Guides\Handlers\RenderCommand;
+
+interface RenderSet
```

Comment:
I'm a bit in doubt about this class name. 

`Renderer` would fit better, but that's too generic. The term `Set` is coming from phpDocumentor. But I'm not sure if it makes sense here. What do you think?